inst/stan/wwinference.stan

```diff
@@ -174,6 +193,18 @@ transformed parameters {
   // per capita infection incidence
   vector[n_subpops] growth_site;
 
+  // Site spatial trans params--------------------------------------------------
+  real phi = exp(log_phi);
```

Comment:
Ah I see, this doesn't conflict because we have called the hospital admissions observation process parameter `phi_h`. Would suggest making a separate issue.